src/instructlab/defaults.py

```diff
@@ -123,15 +126,48 @@ def MODELS_DIR(self) -> str:
 
     @property
     def DEFAULT_CHAT_MODEL(self) -> str:
-        return path.join(self.MODELS_DIR, self.GGUF_MODEL_NAME)
+        return path.join(
+            self.MODELS_DIR,
+            self.HUGGINGFACE_DOMAIN,
+            self.MERLINITE_GGUF_REPO,
+            self.DEFAULT_MODEL_BRANCH,
+        )
+
+    @property
+    def DEFAULT_MODEL_GGUF(self) -> str:
+        return path.join(
+            self.MODELS_DIR,
+            self.HUGGINGFACE_DOMAIN,
+            self.MERLINITE_GGUF_REPO,
+            self.DEFAULT_MODEL_BRANCH,
+        )
+
+    @property
+    def DEFAULT_MODEL(self) -> str:
+        return path.join(
+            self.MODELS_DIR,
+            self.HUGGINGFACE_DOMAIN,
+            self.GRANITE_MODEL_REPO,
+            self.DEFAULT_MODEL_BRANCH,
+        )
 
     @property
     def DEFAULT_TEACHER_MODEL(self) -> str:
-        return path.join(self.MODELS_DIR, self.MISTRAL_GGUF_MODEL_NAME)
+        return path.join(
+            self.MODELS_DIR,
+            self.HUGGINGFACE_DOMAIN,
+            self.MISTRAL_GGUF_REPO,
+            self.DEFAULT_MODEL_BRANCH,
```

Comment:
I feel like there's some duplication here that could be removed? I could be wrong - @cdoern wdyt?

Reply:
removed defaukt_model_gguf, nice catch!